schemas.py

```diff
@@ -0,0 +1,38 @@
+from typing import List
+
+from pydantic import BaseModel
+from datetime import date
+
+
+class BookBase(BaseModel):
+    title: str
+    summary: str
+    publication_date: date
+
+
+class BookCreate(BookBase):
+    pass
+
+
+class Book(BookBase):
+    id: int
+
+    class Config:
+        orm_mode = True
+
+
+class AuthorBase(BaseModel):
+    name: str
+    bio: str
+
+
+class AuthorCreate(AuthorBase):
+    book_id: int
```

Comment:
The `AuthorCreate` schema includes a `book_id` field, which might not be necessary or correct given the one-to-many relationship between authors and books. Typically, the `book_id` would be part of the `Book` model, referencing the `Author`. Consider reviewing the relationship setup in your models to ensure consistency.